aiida/orm/implementation/entities.py

```diff
@@ -0,0 +1,354 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Classes and methods for backend non-specific entities"""
+
```

Comment:
```suggestion
```
Nitpick: I prefer not to have empty lines here. I know there are a lot of those in the codebase, but those were introduced when we automatically removed `from __future__ import print` when removing Python 2 compatibility code. They were followed with an empty line and only the line itself was removed. Writing a script to automatically remove all of them proved trickier than one would think, so I left it for now and am manually removing them whenever I come across one in a file I am editing anyway for a PR.